modules/ROOT/pages/enterprise/export.adoc

```diff
@@ -13,5 +13,5 @@ liveDemo::export[]
 
 :pluginname: Export
 :pluginminimumplan: tiertwo
-:plugindocspage: export
+:plugindocspage: export.adoc
 include::partial$misc/purchase-premium-plugins.adoc[]
```

Comment:
I can't comment on it directly sorry, but the heading below is using the wrong heading level. It should be a `<h2>`, e.g:
```
== Try our Export Demo
```

Reply:
done